docs/configuration.asciidoc

```diff
@@ -14,7 +14,6 @@ The recommended way to configure Elastic APM is to create a file
 
 [source,yaml]
 ----
----
```

Comment:
Not really really related, but I also removed this line in the yml configuration example.